microsoft-edge/web-platform/site-impacting-changes.md

```diff
@@ -21,6 +21,16 @@ For information about upcoming Chromium project web platform changes, see [Chrom
 Check this article often as the Microsoft Edge team updates this article as thinking evolves, timelines solidify, and new changes are announced.
 
 
+<!-- ====================================================================== -->
+## Deprecation of Native Client (NaCl)
```

Comment:
move section into table instead

Reply:
result:
```
| Removal of Native Client (NaCl) | v138 |  | To enhance web security and performance, Native Client (NaCl) is deprecated for Microsoft Edge, along with other browsers.  NaCl was deprecated (discouraged but still supported) starting in 2020.  WebAssembly (Wasm) is recommended instead of NaCl.  See [Native Client (NaCl) (deprecated)](../extensions-chromium/developer-guide/run-compiled-code.md#native-client-nacl-deprecated) in _Run compiled code in an extension_. |
```